coffee/chosen.proto.coffee

```diff
@@ -504,3 +504,14 @@ class @Chosen extends AbstractChosen
         w = f_width - 10
 
       @search_field.setStyle({'width': w + 'px'})
+
+  trigger_form_field_change:  ->
+    Chosen.trigger(@form_field, 'input')
+    Chosen.trigger(@form_field, 'change')
+
+  @trigger: (element, eventType) ->
```

Comment:
Having `trigger` as public method shouldn't be necessary. But this probably has been done so the method is available in the tests as well?